src/main/pages/che-7/contributor-guide/ref_language-server-protocol.adoc

```diff
@@ -13,4 +13,7 @@ summary:
 
 The Language Server Protocol (LSP) defines the protocol used between an editor or IDE and a language server that provides language features, including auto complete, go to definition and find all references. 
 
-For more details, see link:https://microsoft.github.io/language-server-protocol/[].
+For more details, see:
+
+- link:https://microsoft.github.io/language-server-protocol/[Microsoft Langauge Server Protocol].
+- link:https://www.eclipse.org/che/docs/che-7/eclipse-che4z/[Eclipse Che4z]
```

Comment:
@DStatWriter, how is that related?

Reply:
Hi Robert, the Che4z package includes, among other things the Cobol Language support, and some additional functionality to be added, however if you don't feel there is a strong enough link I can remove my added link?

Reply:
It would perhaps make sense to link from the Che4z docs to the LSP content, not vice versa.